Automattic/jetpack
# Boost: remove wp-content/advanced-cache.php when Cache module is disabled

When the Cache module is disabled we should remove the file wp-content/advanced-cache.php so that other caching plugins can be easily used alongside Boost.

ref: [#37227](https://github.com/Automattic/jetpack/issues/37227)

## Proposed changes:

* In Page_Cache.php I added code to handle_module_status_updated() that deletes the advanced-cache.php if the module "page_cache" was disabled.

### Other information:


## Jetpack product discussion


pc9hqz-2QH-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Apply PR
* Enable the Cache module and confirm the file wp-content/advanced-cache.php was created.
* Disable the Cache module and check that wp-content/advanced-cache.php was deleted.
